src/icons/browse.svg

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You should run the svgs through https://jakearchibald.github.io/svgomg/

Reply:
I couldn't save anything appreciable without screwing them up. I can just delete the xml version and comments though.